docs/README.md

```diff
@@ -1,3 +1,61 @@
+- [Go E2E Test Framework for Kubernetes](#go-e2e-test-framework-for-kubernetes)
+- [Design Document](#design-document)
+- [Examples](#examples)
+  - [Leveraging CLI Flags and Adding Custom Flags](#leveraging-cli-flags-and-adding-custom-flags)
+  - [Test Filtering](#test-filtering)
+  - [Using Custom Decoders](#using-custom-decoders)
+  - [Custom Resource Definition](#custom-resource-definition)
+  - [Runtime Features](#runtime-features)
+  - [Multi Cluster Tests](#multi-cluster-tests)
+  - [Third Party Integration](#third-party-integration)
+
+
 # Go E2E Test Framework for Kubernetes
```

Comment:
Hi @harshanarayana this line seems to have moved to line 13. I think It is the title and should be at the top, correct?

Reply:
/Ack. You are right. Let me re order that bit

Reply:
Done. PTAL when possible @vladimirvivien